guides/common/modules/proc_installing-smart-proxy-server-packages.adoc

```diff
@@ -8,12 +8,15 @@ Before installing {SmartProxyServer} packages, you must update all packages that
 .Procedure
 To install {SmartProxyServer}, complete the following steps:
 
```

Comment:
```suggestion
```

Unrelated but still nice: We have a good title and introduction, so I suggest to drop this duplicate line.

Reply:
Dropped